theano/misc/check_blas.py

```diff
@@ -143,6 +143,12 @@ def test():
                   help="The numpy memory layout parameter used when creating"
                   " the numpy.ndarray objects. It accepts 'C' for C memory"
                   " order and 'F' for Fortran order (for all matrices).")
+parser.add_option('--small_only', action='store_true', dest='small_only',
+                  default=False,
+                  help="If true, additionally perform gemm on large matrices.")
+parser.add_option('-B', '--B', action='store', dest='B',
+                  default=5000, type="int",
```

Comment:
I'm not sure I understand. Is your suggestion the same as making the default values 5000 rather than 2000? Or do you want 2000 to still also run?